infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/DefaultConditions.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.service.smartevents.infra.v2.api.models;
+
+import java.util.List;
+
+import com.redhat.service.smartevents.infra.v2.api.models.dto.ConditionDTO;
+
+public final class DefaultConditions {
+
+    public static final String CP_DATA_PLANE_READY_NAME = "DataPlaneReady";
```

Comment:
@lucamolteni "Common" (well defined) names for `Condition`'s available to both `manager` and `operator`.